### ISSUE TITLE
interactive customized geojson layers

### ISSUE THREAD

**Issue by MortenHofft** [gbif/hosted-portals#279]
Via @kpmcfarland - ping @jloomisVCE

The occurrence map  supports adding custom layers. But you cannot interact with them. 

We could look into adding popups with whatever information was added in e.g a geojson layers. And have a convention for how to define a filter for a geometry in geojsons. E.g. `properties: {filterByGeom: true}`. That could then add a popup with free text and a filter button that added the filter to the search.

On first thought this seems doable. But I'm not sure what convetions there might already exist in TileJSON spec for this. That would be nice to knwo before inventing something bespoke

Example layer: https://geodata.vermont.gov/datasets/VTANR::vermont-biophysical-regions/explore?location=43.717365%2C-72.385308%2C8.00

**Comment by jloomisVCE** [comment]
@MortenHofft This would be a terrific feature to add, as Gadm doesn't include subdivisions for eg. Towns, and probably never will for eg. Biophysical regions.

If the ability to filter by any/all GeoJSON properties is too much work, maybe initially offer to search by just one? 
eg. `properties: {filterByGeom: true, filterField:'NAME'}`

**Comment by leo-buitrago** [comment]
I agree it would be a very useful feature to add to the HPs in the future. For National Environmental Ministries and governmental or research institutions to have for instance "Protected Areas" layers would represent a valuable tool.
